### PR TITLE
better readme, better instructions, cleaned up spec

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
@jmburges @avi

### CODE REVIEW

File: spec/hashketball_spec.rb [hunk 1]
```diff
@@ -60,22 +60,11 @@
   describe '#team_colors' do
 
     it 'knows the Brooklyn Nets colors are Black and White' do
-      brooklyn_colors = team_colors("Brooklyn Nets")
-      expect(brooklyn_colors.size).to eq(2)
-      brooklyn_colors.each do |color|
-        expect(["Black", "White"]).to include(color)
+      expect(team_colors("Brooklyn Nets")).to eq(["Black", "White"])
```
Comment (jmburges) [inline spec/hashketball_spec.rb]: Don't use `eq` for arrays. If they return `["White", "Black"]` this won't pass. Use `contain_exactly`

Comment (jmburges) [inline spec/hashketball_spec.rb]: http://www.rubydoc.info/github/rspec/rspec-expectations/RSpec/Matchers#contain_exactly-instance_method

File: spec/hashketball_spec.rb [hunk 2]
```diff
@@ -60,22 +60,11 @@
...
+      expect(team_colors("Charlotte Hornets")).to eq(["Turquoise", "Purple"])
```
Comment (jmburges) [inline spec/hashketball_spec.rb]: bad `eq`

**Comment by jmburges** [comment]
Issues raised. Also this closes issue #2

File: spec/hashketball_spec.rb (hunk 2 above)
Comment (SophieDeBenedetto) [inline spec/hashketball_spec.rb]: fixed

File: spec/hashketball_spec.rb (hunk 1 above)
Comment (SophieDeBenedetto) [inline spec/hashketball_spec.rb]: fixed

**Comment by jmburges** [comment]
:shipit:

**Pull request merged**